README.md

```diff
@@ -14,7 +14,7 @@ Use [docker-compose](http://fig.sh "") and the config from the `docker` folder o
 ```sh
 $ sudo apt-get install docker.io python2 python-virtualenv
 ```
-
+and install [the newest docker](https://docs.docker.com/installation/).
```

Comment:
I have passed this problem before. 
Repo docker.io from apt-get give us the old docker. It will give problem when the docker up, running client and server in different version.
http://askubuntu.com/questions/472412/how-do-i-upgrade-docker